src/configure/helper/templateHelper.ts

```diff
@@ -55,36 +60,47 @@ function isNodeRepo(files: string[]): boolean {
     });
 }
 
+function isFunctionApp(files: string[]): boolean {
```

Comment:
I would rather not have this but always show this template if the language type matches.

Reply:
Why so? package.json for example need not exist for node based Function app

Reply:
IMO we were only doing search for language and showing all resource type templates for that language. So, I thought it's only natural we would do the same for Function App.  (It might also increase the visibility of the template, for it would show for all node.js projects.)

We should not mix two analysis-> language and resource type. We should do this in another iteration, where we should rather 
1. Have two separate functions for language and resource type analysis and then seperate lists for each. 
2. A merge function to join two pipeline lists. with priority.

Reply:
Yes, but as I said, detection logic for node.js is not enough for function app.